tasks/theme.js

```diff
@@ -20,6 +20,10 @@ module.exports = function(grunt) {
    *       "sourcemap": true
    *     }
    *   },
+   *   "legionaire": {
+   *     "path": "<%= config.srcPaths.drupal %>/themes/legionaire",
+   *     "proxy": true
```

Comment:
Should we allow configuration of the grunt command invoked in the theme rather than mandate that they provide a specific command name?